connectors/jetty-connector/src/main/java/org/glassfish/jersey/jetty/connector/JettyConnector.java

```diff
@@ -136,18 +137,28 @@
      * @param config client configuration.
      */
     JettyConnector(final Client jaxrsClient, final Configuration config) {
-        final SSLContext sslContext = jaxrsClient.getSslContext();
-        final SslContextFactory sslContextFactory = new SslContextFactory();
-        sslContextFactory.setSslContext(sslContext);
+        HttpClient httpClient = null;
+        if (config.isRegistered(JettyHttpClientSupplier.class)) {
```

Comment:
@jansupol Seems like you should be using the interface `JettyHttpClientContract` here instead of the implementation?

Reply:
No, it is JettyHttpClientSupplier that is being registered and not the JettyHttpClientContract.